docs/index.md

```diff
@@ -2712,7 +2712,7 @@ less convenient than the built-in paste command provided by metamath-lamp
 (though there's no need to do that here).
 
 You could have done that via a
-long-click on the second parenthesis of the new statement
+click on the second parenthesis of the new statement
```

Comment:
It was in the old versions of mm-lamp that long-click selected fragments by default. In the latest version a short click does this.